spec/features/activities/work_package/activities_spec.rb

```diff
@@ -494,94 +494,89 @@
+        activity_tab.filter_journals(:only_comments)
 
-        # activity_tab.within_journal_entry(latest_journal) do
-        #   activity_tab.expect_journal_notes_header(text: admin.name)
-        #   activity_tab.expect_journal_notes(text: "Third comment by admin")
-        #   activity_tab.expect_no_journal_changed_attribute
-        #   activity_tab.expect_no_journal_details_header
-        # end
+        activity_tab.within_journal_entry(latest_journal) do
+          activity_tab.expect_journal_notes_header(text: admin.name)
+          activity_tab.expect_journal_notes(text: "Third comment by admin")
+          activity_tab.expect_no_journal_changed_attribute
+          activity_tab.expect_no_journal_details_header
+        end
 
-        # activity_tab.filter_journals(:only_changes)
+        activity_tab.filter_journals(:only_changes)
 
-        # activity_tab.within_journal_entry(latest_journal) do
-        #   activity_tab.expect_no_journal_notes_header
-        #   activity_tab.expect_no_journal_notes
+        activity_tab.within_journal_entry(latest_journal) do
+          activity_tab.expect_no_journal_notes_header
+          activity_tab.expect_no_journal_notes
 
-        #   activity_tab.expect_journal_details_header(text: admin.name)
-        #   activity_tab.expect_journal_changed_attribute(text: "Subject")
-        # end
+          activity_tab.expect_journal_details_header(text: admin.name)
+          activity_tab.expect_journal_changed_attribute(text: "Subject")
+        end
       end
 
-      it "resets an only_changes filter if a comment is added by the user", :aggregate_failures do # rubocop:disable RSpec/RepeatedExample
-        pending "works locally but is flaky on CI, reason unknown"
-        fail
-
-        # activity_tab.filter_journals(:only_changes)
-        # sleep 0.5 # avoid flaky test
+      it "resets an only_changes filter if a comment is added by the user", :aggregate_failures do
+        activity_tab.filter_journals(:only_changes)
+        wait_for_filter
 
-        # # expect only the changes
-        # activity_tab.expect_no_journal_notes(text: "First comment by admin")
-        # activity_tab.expect_no_journal_notes(text: "Second comment by admin")
+        # expect only the changes
+        activity_tab.expect_no_journal_notes(text: "First comment by admin")
+        activity_tab.expect_no_journal_notes(text: "Second comment by admin")
 
-        # # add a comment
-        # activity_tab.add_comment(text: "Third comment by admin")
-        # sleep 0.5 # avoid flaky test
+        # add a comment
+        activity_tab.add_comment(text: "Third comment by admin")
+        wait_for_filter
 
-        # # the only_changes filter should be reset
-        # activity_tab.expect_journal_notes(text: "Third comment by admin")
+        # the only_changes filter should be reset
+        activity_tab.expect_journal_notes(text: "Third comment by admin")
       end
+
+      def wait_for_filter = sleep(1)
```

Comment:
this will not be a very stable spec, but I'm willing to merge it and see how flickering it ends up.